lock-api/src/main/java/com/palantir/lock/client/RemoteTimelockServiceAdapter.java

```diff
@@ -41,14 +41,28 @@
 public final class RemoteTimelockServiceAdapter implements TimelockService, AutoCloseable {
     private static final SafeLogger log = SafeLoggerFactory.get(RemoteTimelockServiceAdapter.class);
 
-    private final NamespacedTimelockRpcClient rpcClient;
+    private final NamespacedTimelockTimestampClient rpcClient;
     private final NamespacedConjureTimelockService conjureTimelockService;
     private final LockLeaseService lockLeaseService;
     private final TransactionStarter transactionStarter;
     private final CommitTimestampGetter commitTimestampGetter;
 
+    // TODO(gs): factory method createInMemory?
```

Comment:
I think we decided not to, at least for now? Down to you whether you want to leave this or not